virocon/contours.py

```diff
@@ -16,29 +16,59 @@
            "ISORMContour", "HighestDensityContour", "DirectSamplingContour"]
 
 def calculate_alpha(state_duration, return_period):
+    """
+    Calculates the probability that an observation falls outside the 
+    environmental contour (exceedance probability).
+    
+    Parameters
+    ---------- 
+    state_duration : float
+        Time period for which an environmental state is measured,
+        expressed in hours.
+    return_period : float
+        Describes the average time period between two consecutive 
```

Comment:
This description does not make it clear to me what the return period is or how to obtain it. The formula only defines the threshold, right?